v3/src/components/graph/models/graph-layout.ts

```diff
@@ -96,28 +84,32 @@ export class GraphLayout {
   }
 
   getAxisScale(place: AxisPlace) {
-    return this.axisScales.get(place)
+    return this.axisScales.get(place) ??
+      new MultiScale({scaleType: "ordinal", orientation: "horizontal"})
```

Comment:
Not sure that it makes sense to default to an auto-generated `MultiScale` if there isn't already one present. Did you have a particular use case in mind?